plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/split.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2019 Intel Corporation
```

Comment:
We should be updating copyright on files we're modifying to 2021, not reverting to 2019. Even in cases where we're largely basing the files on older non-PlaidML OpenVINO tests, the combination of those tests with the PlaidML cases is novel work we're doing right now.

Reply:
Done.